elpis/engines/kaldi/objects/model.py

```diff
@@ -236,13 +236,11 @@ def train():
 
                 # Run the command, log output. Also redirect Kaldi sterr output to log. These are often not errors :-(
                 try:
-                    stage_process = run(f"cd {local_kaldi_path}; stages/{stage} >> {stage_log_path}")
-                    with open(stage_log_path, 'a+') as file:
-                        print('stdout', stage_process.stdout, file=file)
-                        print('stderr', stage_process.stderr, file=file)
-                        print('done', file=file)
+                    stage_process = run(f"cd {local_kaldi_path}; stages/{stage} &> {stage_log_path}")
```

Comment:
`&>` ensures both stderr and stdout are logged as they come. This removes the need to print stdout and stderr to file and then print it out.